README.md

```diff
@@ -0,0 +1,79 @@
+# Admission-Trial-Projects
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
Kindly delete the logo 👍

Reply:
- [ ] This comment is polite & clear 👏🏾 . Please add a reason for the requested change. code reviews are also a learning opportunity for the students

Reply:
The logo need to be deleted, you should include your own logo to the project if you have one. 👍